include/eve/detail/function/simd/common/friends.hpp

```diff
@@ -152,7 +176,24 @@ namespace eve::detail
   }
 
   //================================================================================================
-  template<real_simd_value Wide>
+  template<simd_value Wide>
+  EVE_FORCEINLINE auto self_neq(Wide const& v,Wide const& w) noexcept
+  requires( kumi::product_type<element_type_t<Wide>> )
+  {
+    if constexpr( detail::tag_dispatchable<tag::is_not_equal_,Wide,Wide> )
+    {
+      return tagged_dispatch(tag::is_not_equal_{}, v, w);
+    }
+    else
+    {
+      auto zipped = kumi::zip(v.storage(),w.storage());
+      return kumi::fold_right ( [](auto acc , auto e) { return acc || ( get<0>(e) != get<1>(e) ); }
+                              , zipped, false
+                              );
+    }
```

Comment:
I think we should pick up equal customization  for neq

Reply:
We now pick up the tagged_dispatch of is_equal if it is available. For the basic one, I think it's better to do piecewise !=.

Reply:
What about opt in for neq. We don't care?

Reply:
it was removed